### PR TITLE
updated filters for Applicability Filters

### PR DESCRIPTION
**Pull request by deric18**
We recieved a Customer generated CRI about incorrect Description.

### CODE REVIEW

File: memdocs/intune/fundamentals/filters-supported-workloads.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ You can use filters for some common device configuration policies on the followi
 | **Endpoint Security profile** | &nbsp; |
 | Account protection | ✔️ <br/> **Local user group membership** only |
 | Antivirus | ✔️ |
-| Attack surface reduction | ✔️ <br/> Excludes **Web protection (Microsoft Edge Legacy)** and **Application control** |
+| Attack surface reduction | ✔️ <br/> Excludes **Web protection (Microsoft Edge Legacy)** , **Application control** ,**App and browser isolation** and **Device control** |
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/filters-supported-workloads.md:290]:
```suggestion
| Attack surface reduction | ✔️ <br/> Excludes **Web protection (Microsoft Edge Legacy)**, **Application control**, **App and browser isolation**, and **Device control** |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @deric18, kindly commit the suggested changes. Many thanks.

**Comment by ErikjeMS** [comment]
@deric18 If you're a MS employee, please make future changes in the private repo.

#sign-off

**Comment by ErikjeMS** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by MandiOhlinger — APPROVED** [review]

**Comment by MandiOhlinger** [comment]
@deric18 Thank you!

**Pull request merged**